core/src/main/resources/hudson/model/Job/configure.jelly

```diff
@@ -59,8 +59,8 @@ THE SOFTWARE.
             <j:if test="${h.hasPermission(it,it.CONFIGURE)}">
               <f:bottomButtonBar>
                 <!--<input type="button" name="StructureTest" value="Test" onclick="buildFormTree(this.form)" />-->
-                <f:submit value="${%Save}" large="true"/>
-                <f:apply value="${%Apply}" large="true" />
```

Comment:
What happens in views that currently set this attribute? Is it just an obsolete no-op?

Reply:
The attribute is ignored 👍